components/features/github/github-activity.tsx

```
@@ -0,0 +1,202 @@
+          Loading activity...
+        </div>
+      );
+    }
+    if (error) {
+      return (
+        <div
+          onClick={navigateToGitHub}
+          className="block text-center text-red-500 hover:text-red-400 transition-colors cursor-pointer"
+          aria-label="View William Callahan's GitHub profile"
+          title="View William Callahan's GitHub profile"
+          role="button"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<button>
<input type="button">


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)